app/views/categories/_form.html.erb

```diff
@@ -1,46 +1,49 @@
 <%# locals: (category:, categories:) %>
+<%= turbo_frame_tag 'form' do %>
```

Comment:
Do we need this?  What was the purpose of adding?

Reply:
ohhh, i forgot to remove this... it's done in [2c07cd6](https://github.com/maybe-finance/maybe/pull/1730/commits/2c07cd6d19ee5d43e9e4fb1a36e4ed49b3702d89). ;-)